src/goto-instrument/contracts/contracts.cpp

```diff
@@ -1260,3 +1259,34 @@ bool code_contractst::enforce_contracts(const std::set<std::string> &to_enforce)
   }
   return fail;
 }
+
+void code_contractst::find_static_locals(
+  const goto_functionst &functions,
+  const irep_idt &root_function,
+  assigns_clauset &assigns)
+{
+  auto cg =
+    call_grapht::create_from_root_function(functions, root_function, true);
+  auto g = cg.get_directed_graph();
```

Comment:
Why don't we hit this case?

Reply:
I'm curious too. Shouldn't we hit this on `regression/contracts/assigns_enforce_statics_subfunctions` test?

Reply:
This method is indeed covered when running  `regression/contracts/assigns_enforce_statics_subfunctions` otherwise the test would not pass.

Reply:
Thanks. Probably a false positive / stale result from Codecov then. Resolving this.